pkg/resource/v1/certconfig/update.go

```diff
@@ -7,12 +7,37 @@ import (
 	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
+	"k8s.io/api/core/v1"
 )
 
+// ApplyUpdateChange takes observed custom object and update portion of the
+// Patch provided by NewUpdatePatch or NewDeletePatch.
 func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange interface{}) error {
+	certConfigsToUpdate, err := toCertConfigs(updateChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if len(certConfigsToUpdate) > 0 {
+		r.logger.LogCtx(ctx, "level", "debug", "message", "updating certconfigs")
+
+		for _, certConfigToUpdate := range certConfigsToUpdate {
+			_, err = r.g8sClient.CoreV1alpha1().CertConfigs(v1.NamespaceDefault).Update(certConfigToUpdate)
```

Comment:
I think it would be good to use the namespace of the actual object.

Reply:
Yes, this is something I asked earlier already. Currently all `CertConfigs` are in `default` namespace. I also believe this shouldn't be the case, but for the transition period from `kubernetesd` -> `cluster-operator` I think it's good to have them in old location and then maybe create a migration path to `NewUpdatePatch()`. WDYT?

Reply:
Ok. Discussed in Meet about this. Namespace here should be the same as with custom object. I'll fix this in a separate PR as this same pattern is present elsewhere too.